frontend/app/components/card.tsx

```diff
@@ -0,0 +1,161 @@
+import type { ComponentProps, HTMLElementType, KeyboardEvent, MouseEvent } from 'react';
+
+import type { To } from 'react-router';
+
+import type { FlipProp, IconProp } from '@fortawesome/fontawesome-svg-core';
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { AppLink } from '~/components/links';
+import type { I18nRouteFile } from '~/i18n-routes';
+import { cn } from '~/utils/tailwind-utils';
+
+type OptionalAppLink = Omit<ComponentProps<typeof AppLink>, 'to' | 'lang'> & {
```

Comment:
I don't think you have to omit `lang` here.

Reply:
Also nitpick: the convention for naming prop types is to add a `Props` suffix (ex: `CardBaseProps`)